doc/release-notes.md

```diff
@@ -93,6 +93,11 @@ Low-level RPC changes
   now the empty string `""` instead of `"wallet.dat"`. If bitcoin is started
   with any `-wallet=<path>` options, there is no change in behavior, and the
   name of any wallet is just its `<path>` string.
+- Passing an empty string (`""`) as the `address_type` parameter to
+  `getnewaddress`, `getrawchangeaddress`, `addmultisigaddress`,
+  `addmultisigaddress`, `fundrawtransaction` RPCs is now an error. Previously,
```

Comment:
nit: Is there any difference between "addmultisigaddress" and "addmultisigaddress"?